contracts/src/LightClient.sol

```diff
@@ -282,12 +285,17 @@ contract LightClient is Initializable, OwnableUpgradeable, UUPSUpgradeable {
 
     /// @dev Simple getter function for the genesis state
     function getGenesisState() public view virtual returns (LightClientState memory) {
-        return states[genesisState];
+        return genesisState;
+    }
+
+    /// @dev Simple getter function for the genesis stake state
+    function getGenesisStakeState() public view virtual returns (StakeState memory) {
+        return genesisStakeState;
     }
 
     /// @dev Simple getter function for the finalized state
     function getFinalizedState() public view virtual returns (LightClientState memory) {
-        return states[finalizedState];
+        return finalizedState;
     }
```

Comment:
remove all of these helpers because these states are trivial to read (and come with default getter)

Reply:
ah yes, it was used when we had the mapping, updated here https://github.com/EspressoSystems/espresso-sequencer/commit/56b88cd34eeab65ce00ba3e8a954accfc850937f